pages/contribute.jsx

```diff
@@ -1,13 +1,225 @@
-// TODO: this page will be replaced by new contribute page
+import React from 'react';
+import Image from 'next/image';
+import Link from 'next/link';
 
-import Contribute from 'components/ContributePage';
 import PageWrapper from 'components/Layout/PageWrapper';
 import Meta from 'components/Meta';
+import {
+  MAIN_TITLE_CLASS,
+  TEXT_SMALL_CLASS,
+  SECTION_BOX_CLASS,
+  TEXT_MEDIUM_LIGHT_CLASS,
+  TEXT_CLASS,
+  SUB_HEADER_CLASS,
+} from 'common-util/classes';
+import SectionWrapper from 'components/Layout/SectionWrapper';
+import { Button } from 'components/ui/button';
+
+const HeroImage = () => (
+  <Image
+    src="/images/contribute-page/hero.png"
+    alt="Contribute Hero"
+    width={340}
+    height={350}
+  />
+);
+
+const Hero = () => (
+  <SectionWrapper customClasses={`border-b ${SECTION_BOX_CLASS}`}>
+    <div className="grid max-w-screen-xl items-start mx-auto lg:px-12 lg:gap-8 lg:grid-cols-12 lg:items-top xl:gap-0">
+      <div className="md:mb-12 lg:col-span-6">
+        <div className={`${TEXT_MEDIUM_LIGHT_CLASS} mb-2`}>CONTRIBUTE</div>
+        <h2 className={`mb-4 ${MAIN_TITLE_CLASS}`}>
+          Track community impact, transform your network
+        </h2>
+        <div className="md:hidden mb-8">
+          <HeroImage />
+        </div>
+        <div className={TEXT_SMALL_CLASS}>
+          Leverage autonomous services to track and enhance the influence of
+          your community.
+        </div>
+        <Button variant="default" size="xl" className="mt-6 w-full md:w-auto">
```

Comment:
can we pass `asChild` prop to the button? so the whole buttons is a link, but not only the text